README.md

```diff
@@ -17,3 +17,14 @@ any good:
   in the image.
 * Running `go test -c` and running the test binary under `docker run` is
   tedious, error-prone, and doesn't scale to many packages.
+
+### Caveats
+
+* `go test` without package arguments runs tests with access to the current
+  terminal. However, `go test -exec="dockexec $image"` will not, as `dockexec`
+  cannot distinguish this mode from others like `go test -exec="dockexec $image"
+  ./...`. If you want access to the terminal, supply the `-i -t` docker flags.
```

Comment:
Hmm I'm not sure whether `-i` makes any sense, especially if you have multiple packages being tested concurrently.

Reply:
I only added it because, technically, one could run `go test` in a way that the test is actually interactive with stdin. But I really hope noone is doing that :) So yeah, I'll remove that.